lifetimes/fitters/beta_geo_beta_binom_fitter.py

```diff
@@ -68,7 +69,7 @@ def _sum(x, tx, recency_T):
             if recency_T <= -1:
                 return -np.inf
 
-            j = J[:recency_T + 1]
+            j = J[:int(recency_T) + 1]
```

Comment:
can you explain this change?

Reply:
cool just wanted to ping you again to see if you approve of this change or not